stdlib/log/logger.lua

```diff
@@ -76,6 +79,9 @@ function Logger.new(mod_name, log_name, debug_mode, options)
                 return Logger.write()
             end
         else
+            if Logger.options.use_log then
```

Comment:
It seems like there is a better way to detect this than a new flag. If `_G.script` does not exist, then we're not in control.lua.